src/main/java/edu/project4/utils/Correction.java

```diff
@@ -0,0 +1,41 @@
+package edu.project4.utils;
+
+import edu.project4.FlameImage;
+import edu.project4.Pixel;
+import java.awt.Color;
+import static java.lang.Math.log10;
+import static java.lang.Math.pow;
+
+public final class Correction {
+    private Correction() {
+    }
+
+    public static void correct(FlameImage image, double gamma) { //чем меньше значение, тем темнее
+        double max = 0.0;
```

Comment:
Предложение: избежать деления на ноль.

```suggestion
        double max = Double.MIN_VALUE;
```